controllers/project_reconciler.go

```diff
@@ -15,6 +15,18 @@ import (
 	"sigs.k8s.io/controller-runtime/pkg/reconcile"
 )
 
+var projectlabels = map[string]string{
+	"app.kubernetes.io/part-of": "project",
+}
+
+const (
+	PROJECTROLEBINDINGUSERNAME     = "edit"
+	PROJECTROLEBINDINGKINDUSERNAME = "ClusterRole"
+	PROJECTSERVICEACCOUNTNAME      = "default"
+	PROJECTROLEBINDINGSAROLENAME   = "view"
```

Comment:
EDITCLUSTERROLENAME

Reply:
i will change variable PROJECTROLEBINDINGSAROLENAME to variable `VIEW_CLUSTER_ROLE_NAME`